cypress/integration/2.3-verify_pickup_points.spec.js

```diff
@@ -7,7 +7,8 @@ describe('Verify all pickup points is displayed', () => {
   testSetup()
 
   listallPickupPointsAPI()
-  it('Verify all pickup points is shown in stores', updateRetry(2), () => {
+  it('Verify all pickup points is shown in stores', updateRetry(5), () => {
```

Comment:
Soft retry can be 2
Hard retry can be increase to 3/4 based on your needs

+

Add information in changelog + description in PR ( So, that security check can be passed)